packages/eventual-send/src/local.js

```diff
@@ -13,6 +18,55 @@ const ntypeof = specimen => (specimen === null ? 'null' : typeof specimen);
  */
 const isObject = val => Object(val) === val;
 
+const BREAKPOINTS = getEnvironmentOption('BREAKPOINTS', '');
+
+/**
+ * @type {Record<string, [string, (number | '*')][]>}
+ */
+// @ts-expect-error doesn't like null __proto__
+const BREAKS = { __proto__: null };
+
+const BP_REGEXP = /(.+?)\.(\w+?|\*)=(\d+?|\*)/;
+
+for (const bp of BREAKPOINTS.split(',')) {
+  if (bp !== '') {
+    const match = BP_REGEXP.exec(bp);
+    if (match === null) {
+      throw Fail`Expected "tagName.methodName=count" ${q(bp)}`;
+    }
+    const [_, tag, verb, countStr] = match;
+    const patts = hasOwn(BREAKS, verb) ? BREAKS[verb] : (BREAKS[verb] = []);
+    patts.push([tag, countStr === '*' ? countStr : +countStr]);
+  }
+}
+
+const debugApply =
+  BREAKPOINTS === ''
+    ? (f, obj, _verb, args) => apply(f, obj, args)
+    : (f, obj, verb, args) => {
+        const patts = BREAKS[verb] || BREAKS['*'];
+        if (patts) {
+          for (const patt of patts) {
+            const [tag, count] = patt;
+            if (tag === '*' || tag === obj[Symbol.toStringTag]) {
+              if (count === '*') {
+                // eslint-disable-next-line no-debugger
+                debugger;
+                break;
+              } else {
+                if (count === 0) {
+                  // eslint-disable-next-line no-debugger
+                  debugger;
+                  break;
+                }
+                patt[1] = count - 1;
+              }
+            }
+          }
+        }
+        return apply(f, obj, args);
```

Comment:
Is there a benefit to this `Record<Verb, Array<[Tag, Count]>>` structure over `Record<Verb, Record<Tag, Count>>`?
```suggestion
        const countsByTag = BREAKS[verb] || BREAKS['*'];
        if (countsByTag !== undefined) {
          const objTag = obj[Symbol.toStringTag];
          const matchTag = hasOwn(countsByTag, objTag) ? objTag : '*';
          const count = countsByTag[matchTag];
          if (count === '*' || count === 0) {
            // eslint-disable-next-line no-debugger
            debugger;
          } else if (count !== undefined) {
            countsByTag[matchTag] = count - 1;
          }
        }
        return apply(f, obj, args);
```

Reply:
Done